kim_cnn/README.md

```diff
@@ -10,11 +10,47 @@ Implementation for Convolutional Neural Networks for Sentence Classification of
 - multichannel: A model with two sets of word vectors. Each set of vectors is treated as a 'channel' and each filter is applied to both channels, but gradients are back-propagated only through one of the channels. Hence the model is able to fine-tune one set of vectors while keeping the other static. Both channels are initialized with word2vec.# text-classification-cnn
 Implementation for Convolutional Neural Networks for Sentence Classification of [Kim (2014)](https://arxiv.org/abs/1408.5882) with PyTorch.
 
+## Requirement
+
+- Install the latest version of pytorch
+- Install the version of torchtext
+```
+git clone https://github.com/pytorch/text.git 
+cd text  
+#use this commit number 
+git reset --hard 6f930eb8de936482f321ac123e020fd08dd07a4b
```

Comment:
Would it be better to use major released that are published on PyPI? This way users can install using pip instead of running setup from source.

Currently, torchtext release v0.2.0 is available: https://pypi.python.org/pypi/torchtext. This is pretty recent.